### DISCUSSION TITLE
why coal farm flat?

### DISCUSSION THREAD

**Discussion by mricebear** [voxeltycoon/issues#875] [General] [answered]
hello, im new player.

anyway why coal farm is flat tile in my game?
![Untitled](https://user-images.githubusercontent.com/87141731/124957257-15511c80-e043-11eb-9f18-817befb2aac8.png)

**Comment by SlightlyAmazing** [comment, marked as answer]
The black area is not a coal mine, it is signifying a coal resource.
The 'mine' is the building you have placed, the one is the left - centre of the picture. That has a pully system (sticking out the top of it) that would lead into deeper Cavens and tunnels

**Comment by mricebear** [comment]
sry wrong question, why coal resource is plat tile?
should insert some good building
